gitissues.widget/index.jsx

```diff
@@ -113,16 +113,22 @@ export const updateState = (event, previousState) => {
   }
   // What we need now is to fetch the issues and display them.
   let current = new Date()
-  let lastChecked = 'Last updated on ' + `${monNames[current.getMonth()]} ${current.getDate()}, ${current.getFullYear()}, ${current.getHours()}:${current.getMinutes()}`
+  let lastChecked = 'Last updated on ' + `${monNames[current.getMonth()]} ${current.getDate()}, ${current.getFullYear()}, ${current.getHours()}:${String(current.getMinutes()).padStart(2, "0")}`
   // Reset the issues to overwrite old ones
   previousState.displayIssues = []
   for (let issue of event.data) {
-    if (issue.state === 'open') {
+    if (issue.state === 'open') { 
       let t = Date.parse(issue.updated_at)
-      if (Date.now() - t < 86400000) {
-        t = 'yesterday'
-      } else if (Date.now() - t < 604800000) {
-        t = 'last week'
+      let now = Date.now()
+      let elapsed = now - t
+      if (elapsed < 86400000) {
```

Comment:
```suggestion
      if (elapsed < 86_400_000) {
```

With these large numbers, underscores are in order.

Reply:
I wasn't aware this syntax is available...just learned something new. I initially converted everything to seconds, but switched back to match your original code.